### PR TITLE
@curve/create lockevent

### PR DESCRIPTION
**Pull request by yasmine9**

### CODE REVIEW

File: Curve-finance-suite/Create-lockevent/src/agent.ts [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    )
+    return Finding.fromObject({
+      name: "Lock creation on Voting Escrow contract Detected",
+      description: "Deposit event with type 1 was emitted",
+      alertId: alertID,
+      severity: FindingSeverity.Low,
+      type: FindingType.Suspicious,
+      protocol: 'Curve Finance',
+      metadata: {
+        from: metadata!.topics[1],
```
Comment (mauricio1802) [inline Curve-finance-suite/Create-lockevent/src/agent.ts:31]: Can we decode this as an `address` for making the finding information more readable?

File: Curve-finance-suite/Create-lockevent/README.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+## Alerts
+
+- CURVE-7
+  - Fired when a Deposit event is emitted with type set to 1.
+  - Severity is always set to "low".
+  - Type is always set to "suspicious".
+  - Metadata includes: 
+    - `from`: address of the account calling create lock function.
+    - `value`: The amount of CRV locked.
+    - `locktime`: lock duration.
```
Comment (mauricio1802) [inline Curve-finance-suite/Create-lockevent/README.md:20]: Can we check if this is the `lock duration` or `when the lock ends`

File: Curve-finance-suite/Create-lockevent/src/agent.spec.ts [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+const ALERT_ID = 'Create lock Agent test';
+const SENDER_ADDRESS = createAddress('0x5');
+const EVENT_SIGNATURE = 'Deposit(address,uint256,uint256,int128,uint256)';
+
+const createFinding = (from: string, value:string, locktime: string ) => Finding.fromObject({
+  name: 'Lock creation on Voting Escrow contract Detected',
+  description: 'Deposit event with type 1 was emitted',
+  alertId: ALERT_ID,
+  severity: 2,
+  type: 2,
```
Comment (mauricio1802) [inline Curve-finance-suite/Create-lockevent/src/agent.spec.ts:27]: Can we use the constants from `forta-sdk` for making this more readable?
`FindingSeverity.Low` and `FindingType.Suspicious`

File: Curve-finance-suite/Create-lockevent/src/agent.spec.ts [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+  metadata: {
+    from: from,
+    value: value,
+    locktime: locktime,
+  },
+});
+
+describe('Create Lock for Voting Escrow contract tests suite', () => {
+  let handleTransaction: HandleTransaction = provideCreateLockAgent(ALERT_ID, CONTRACT_ADDRESS);
+  const topics: string[] = [ SENDER_ADDRESS, encodeParameter('int128', 1764806400) ];
```
Comment (mauricio1802) [inline Curve-finance-suite/Create-lockevent/src/agent.spec.ts:38]: Can we encode `SENDER_ADDRESS` too for making the test case more realistic?

**Review by CeliktepeMurat — APPROVED** [review]

File: Curve-finance-suite/Create-lockevent/README.md (hunk 2 above)
Comment (yasmine9) [reply Curve-finance-suite/Create-lockevent/README.md]: Ah yes it's the time when to unlock the CRV tokens. Readme updated.

**Review by mauricio1802 — APPROVED** [review]

**Pull request merged**